extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/Traceless.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.opentelemetry.runtime.tracing;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Identifies that the current path should not be select for tracing.
+ * <p>
+ * Used together with {@code jakarta.ws.rs.Path} annotation.
```

Comment:
Can you please add a test where a `@Traceless` is used outside of a rest endpoint? 
I imagine you get an exception, would it be possible to have a nice message in that case?

Reply:
We do not have an exception, should we add an exception?

Reply:
maybe a warning

Reply:
If this only makes sense for rest endpoints, yes.